modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -0,0 +1,25 @@
+= Activity Tracking
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+B2B transactions often communicate contractual obligations between companies. It is therefore desirable (and often necessary) to track transmissions sent or received and store the transmission content as well as metadata. Metadata could be the date and time the transmission was sent or received, the identity of the partner, and the success or failure of processing. Anypoint Partner Manager provides a simple, secure, and integrated mechanism for activity tracking within MuleSoft flows. Some uses for this data include:
+
+* Non-repudiation.
+* Correlation of acknowledgments.
+* Error correction.
+* Auditing.
+
+Anypoint Partner Manager enables you to monitor B2B activity tracking. Activity is the full set of end-to-end events (including transmissions, documents, and errors) involved in processing a transmission.
+
+You can:
+
+* Filter to find the specific transmission data (for Beta?).
+* Review and monitor errors and acknowledgements.
+* Search for document metadata (for Beta?).
+
+== To Track Activity:
+
+. Select Activity from the left hand navigation. You see the activity on the message flows you created. +
+. Scroll the list. If you don't see the desired message flow, type it's name in the search box. +
+. Select the message flow. You see... +
```

Comment:
???

Reply:
That's all that works right now. Wasn't sure how to show that. Perhaps with comments.